index.html

```diff
@@ -5,12 +5,95 @@
+    />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
-    <!-- Add your markup here -->
-</body>
+    <header>
+        <nav class="container container-flex header">
+          <img src="../images/sumo1.jpg" alt="sumo" class="logo" />
+          <label for="check" class="check-btn">
+            <i class="fa fa-bars"><p class="checkbtn-text">menu</p></i>
+          </label>
+  
+          <h2 class="shop-title">sumo bakery</h2>
+          <input type="checkbox" id="check" />
+          
+          <ul class="nav-bar nav-list">
+            <li class="nav-list-item"><a href="" class="nav-link">home</a></li>
+            <li class="nav-list-item"><a href="" class="nav-link">order</a></li>
+            <li class="nav-list-item"><a href="#service" class="nav-link">service</a></li>
+            <li class="nav-list-item"><a href="" class="nav-link">cake</a></li>
+            <li class="nav-list-item"><a href="" class="nav-link">feedback</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      
+          
+          <section id="service"  class="container container-flex hero-area"> 
+            
+              <aside >
+                  <h1>HELLO</h1>
+                  <p> Lorem ipsum, <strong>SUMO like cake a lot </strong>  dolor sit amet
+                     consectetur adipisicing elit. Eveniet, provident excepturi? Illum veritatis, 
+                     ullam, amet vero debitis maiores velit ipsa sit non, neque itaque eius quibusdam 
+                     consequatur in incidunt tenetur quia. Id et deleniti doloribus modi quas ipsum esse nam. 
+                    Lorem ipsum do <strong>He make Cake with ......</strong>, amet consectetur adipisicing elit. Reprehenderit
+                     molestiae    praesentium tempore saepe eos, quo eligendi nobis. Nemo, magnam ut. <a  class="article-info"  href=""> Read More </a></p>
+                  
+              </aside>
+              <main>
+                
+                <article class= "article-recent  hero-cake">
+                  <img class="display-cake" src="../images/cake1.jpg" alt="cake">
+
+                      
+                  <img src="../images/cake2.jpg" alt="cake">
+                  <img src="../images/cake3.jpg" alt="cake">
+                  <img src="../images/cake4.png" alt="cake">
+                  <img src="../images/cake5.jpg" alt="cake">
+              </article>  
+                
+              
+              </main>
+          </section id="service">
+          <section class="container container-flex service-area">
+            <article class="article-recent service">
+              <img src="../images/bakery1.jpg" >
+              <img src="../images/bakery2.jpg" >
+              <img src="../images/bakery3.jpg" >
+              <img src="../images/bakery4.jpg" >
+            </article>
+            
+          </section>
```

Comment:
Why is this section outside of `main`? If it's not inside the header or the footer, keep it in main.

Reply:
than you again . i will add change to main instead of section.